src/hooks/useDydxClient.tsx

```diff
@@ -57,6 +59,11 @@ const useDydxClientContext = () => {
   const [compositeClient, setCompositeClient] = useState<CompositeClient>();
   const [faucetClient, setFaucetClient] = useState<FaucetClient>();
 
+  // assume there's only one option for indexer endpoints
+  const indexerEndpoints = ENVIRONMENT_CONFIG_MAP[selectedNetwork].endpoints.indexers[0];
```

Comment:
Can we abstract `endpoints` into it's own hook like we do for `tokenConfigs` or `URLConfigs`?